apps/web/src/components/submission/components/Preferences.tsx

```diff
@@ -80,6 +94,37 @@ export const Preferences: React.FC<FormStepProps> = () => {
         legend='Have you previously been deployed from the EHPR?'
         options={previousDeploymentOptions}
       />
+      {values.preferencesInformation.hasPreviousDeployment === 'yes' && (
+        <>
+          <div className='flex flex-row'>
+            <DatePickerField
+              name='preferencesInformation.lastDeploymentDate'
+              label='Date of Last Deployment'
+              format='yyyy-MM-dd'
+              max={new Date()}
+            />
+          </div>
+          <div className='flex flex-row'>
+            <Field
+              name={'preferencesInformation.lastDeployedHa'}
+              label={'Last Deployed Health Authority'}
+              component={({ field, form }: FieldProps) => (
+                <ReactSelect<OptionType>
```

Comment:
Looks like this is getting flagged by sonar cloud

Reply:
seems to be an issue with the select being nested, I looked through the other issues in sonarcloud and it seems to flag all occurrences of this ReactSelect, I shall look into it